zombies.js

```diff
@@ -293,7 +448,12 @@
  * -----------------------------
  */
 
-
+class RangedZombie extends Zombie{
+  constructor(health, strength, speed){
+    super(health, strength, speed);
+    this.speed = speed;
```

Comment:
speed is part of the super constructor, so I does not need to be set on the child